org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,201 @@
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rhboot/efivar/releases/download/35/efivar-35.tar.bz2",
+                    "sha256": "1e033dc5d099a44fd473b0887dbcc4b105613efab0fb3c5df9f111ea5d147394"
+                }
+            ]
+        },
+        {
+            "name": "libsmbios_c",
+            "config-opts": ["--disable-doxygen",
+                            "--disable-graphviz",
+                            "--disable-python"],
+            "cleanup": [
+                "/sbin/smbios*",
+                "/share/locale/*/LC_MESSAGES/libsmbios.mo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/dell/libsmbios/archive/v2.4.2.tar.gz",
+                    "sha256": "ebfe18415e24bbec06d0a9ea1066c8dcd82982555373712713d7e194138650de"
+                }
+            ]
+        },
+        {
+            "name": "gnu-efi",
+            "buildsystem": "simple",
+            "build-commands": ["make", "make PREFIX=/app install"],
+            "no-autogen": true,
+            "cleanup": [
+                "/bin/efivar"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://superb-dca2.dl.sourceforge.net/project/gnu-efi/gnu-efi-3.0.5.tar.bz2",
+                    "sha256": "bd8fcd5914f18fc0e4ba948ab03b00013e528504f529c60739b748f6ef130b22"
+                }
+            ]
+        },
+        {
+            "name": "python3-pillow",
+            "buildsystem": "simple",
+            "build-commands": ["pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} ."],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/93/73/66854f63b1941aad9af18a1de59f9cf95ad1a87c801540222e332f6688d7/Pillow-4.1.1.tar.gz",
+                    "sha256": "00b6a5f28d00f720235a937ebc2f50f4292a5c7e2d6ab9a8b26153b625c4f431"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "python3-olefile",
+                    "buildsystem": "simple",
+                    "build-commands": ["pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} ."],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://pypi.python.org/packages/35/17/c15d41d5a8f8b98cc3df25eb00c5cee76193114c78e5674df6ef4ac92647/olefile-0.44.zip",
+                            "sha256": "61f2ca0cd0aa77279eb943c07f607438edf374096b66332fae1ee64a6f0f73ad"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "fwupd",
+            "buildsystem": "meson",
+            "config-opts": ["-Dconsolekit=false",
```

Comment:
Just for consistency formatting is:

```
"foo": [
  "bar"
]
```

Reply:
Fixed, thanks. I also switched everything to 2-space indents rather than the crazy mix there was before.